frontend/src/app/main/ui/viewer.cljs

```diff
@@ -255,6 +257,8 @@
     (when (not allowed)
       (st/emit! (dv/go-to-section :interactions)))
 
+    (st/emit! (modal/hide))
```

Comment:
Do not emit events on body of event, you want perform side effects, it is better to wrap it in an effect hook:

```clojure
(mf/with-effect []
  (st/emit! (modal/hide)))
```

Reply:
The same for the previous code that is not part of this PR.